keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -524,11 +524,201 @@ minimum kubelet version, maximum kubelet version, and enabled featureset version
 
 #### Properties
 
-Name/value pairs to represent properties of the clusters. It could be a
-collection of ClusterProperty resources, but could also be info based on
-other implementations. The name of the cluster property can be predefined
-name from ClusterProperty resources and is allowed to be customized by
-different cluster managers.
+Properties defines cluster characteristics through a list of Property objects.
+Each Property can be one of:
+ 1. A ClusterProperty resource (as defined in KEP-2149)
+ 2. Custom information from cluster manager implementations
+
+The field contains a name/value pair and the last observed time of the property on the cluster the ClusterProfile object represents.
+
+For a ClusterProperty resource to be propagated to a ClusterProfile object, it MUST have a label with key "multicluster.x-k8s.io/clusterProfile" and value of "true".
+
+The following are standard ClusterProperty resources that the ClusterProfile API supports:
+
+##### cluster-entrypoints.k8s.io
+This property contains a list of hostname:port or IP:port pairs that can be used to reach the API server of the Kubernetes cluster.
+  
+Here is an example of a cluster-entrypoints.k8s.io ClusterProperty
+```yaml
+apiVersion: about.k8s.io/v1
```

Comment:
I keep getting confused about it. The [code](https://github.com/kubernetes-sigs/about-api/blob/master/clusterproperty/api/v1alpha1/groupversion_info.go#L20) is still v1alpha1, there is a open PR for [v1beta1](https://github.com/kubernetes-sigs/about-api/pull/27) and [docs](https://multicluster.sigs.k8s.io/concepts/about-api/) are v1.

Reply:
my bad, I think it's still v1alpha1